samples/plugins/mysql_odbc/manifest.xml

```diff
@@ -4,6 +4,7 @@
   <vendor-information>
     <company name='TestCompany' />
     <support-link url='http://example.com' />
+    <driver-download-link url="http://driverexamples.com"/>
```

Comment:
Use `http://drivers.example.com`.  It will 404, but I think thats ok and I'd rather not use random domains in the samples.

Reply:
yeah, makes sense. Fixed.